src/Symfony/Bundle/FrameworkBundle/Command/CachePoolGetSetCommand.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Get and/or set cache pool key value.
+ *
+ * @author Antoine Makdessi <amakdessi@me.com>
+ */
+final class CachePoolGetSetCommand extends Command
+{
+    protected static $defaultName = 'cache:pool:get-set';
+
+    private $pools;
+
+    public function __construct(array $pools)
+    {
+        parent::__construct();
+
+        $this->pools = $pools;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Allow to get and/or set key value from cache pools.')
+            ->addArgument('pool', InputArgument::REQUIRED, 'The cache pool from which to get/set a key')
+            ->addArgument('key', InputArgument::REQUIRED, 'The cache key to get/set')
+            ->addArgument('value', InputArgument::OPTIONAL, 'The value to set in the cache pool key')
+            ->addOption('save', null, InputOption::VALUE_OPTIONAL, 'Save the value inside the key of the pool', false)
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> get a value from a given cache pool key.
+
+    %command.full_name% <pool> <key>
+
+The <info>%command.name%</info> set a value on a key on a given cache pool.
+
+    %command.full_name% <pool> <key> <value> --save
```

Comment:
Why requiring the `--save` option? Providing the second value should be enough.

If it's for safety, what about two separated `get/set` commands? This would also allow to get multiple cache keys at once by providing as many keys you want as arguments.

Reply:
> Providing the second value should be enough.

indeed, i will remove it 👍 

> what about two separated

at first i was about to make 2, but as the feature is very close i went for one
lets wait other reviews :)